test/suites/zombie_tanssi_relay_keystore/test_zombie_tanssi_relay_keystore.ts

```diff
@@ -0,0 +1,411 @@
+
+            collator01KeystorePath = `${getTmpZombiePath()}/Collator-01/relay-data/chains/dancelight_local_testnet/keystore/`;
+            // Collator-02 keystore is in a different path because we have added a custom `--keystore-path` arg
+            collator02KeystorePath = `${getTmpZombiePath()}/Collator-02/relay-data/tmp_keystore_zombie_test/`;
+        }, 120000);
+
+        it({
+            id: "T01",
+            title: "Test block numbers in relay are 0 yet",
+            test: async () => {
+                const header2000 = await getHeaderFromRelay(relayApi, 2000);
+                const header2001 = await getHeaderFromRelay(relayApi, 2001);
+                const header2002 = await getHeaderFromRelay(relayApi, 2002);
+
+                expect(header2000.number.toNumber()).to.be.equal(0);
+                expect(header2001.number.toNumber()).to.be.equal(0);
+                expect(header2002.number.toNumber()).to.be.equal(0);
+            },
+        });
+
+        it({
+            id: "T02",
+            title: "Blocks are being produced on tanssi-relay",
+            test: async () => {
+                const relayNetwork = relayApi.consts.system.version.specName.toString();
+                expect(relayNetwork, "Relay API incorrect").to.contain("dancelight");
+                const blockNum = (await relayApi.rpc.chain.getBlock()).block.header.number.toNumber();
+                expect(blockNum).to.be.greaterThan(0);
+            },
+        });
+
+        it({
+            id: "T02b",
+            title: "Collator-01 keystore path exists",
+            test: async () => {
+                collator01KeystoreLength = await countFilesInKeystore(collator01KeystorePath);
+            },
+        });
+
+        it({
+            id: "T02c",
+            title: "Collator-02 keystore path exists",
+            test: async () => {
+                collator02KeystoreLength = await countFilesInKeystore(collator02KeystorePath);
+            },
+        });
+
+        it({
+            id: "T03",
+            title: "Set config params",
+            test: async () => {
+                const keyring = new Keyring({ type: "sr25519" });
+                const alice = keyring.addFromUri("//Alice", { name: "Alice default" });
+
+                // Disable rotation
+                const tx1 = relayApi.tx.collatorConfiguration.setFullRotationPeriod(0);
+                const fillAmount = 990_000_000; // equal to 99% Perbill
+                const tx2 = relayApi.tx.collatorConfiguration.setMaxParachainCoresPercentage(fillAmount);
+                const txBatch = relayApi.tx.utility.batchAll([tx1, tx2]);
+                await signAndSendAndInclude(relayApi.tx.sudo.sudo(txBatch), alice);
+            },
+        });
+
+        it({
+            id: "T04",
+            title: "Test assignation did not change",
+            test: async () => {
+                const currentSession = (await relayApi.query.session.currentIndex()).toNumber();
```

Comment:
We had already something similar in the previous test.
Maybe we want to extract these first 10 steps to the separate suite and run it for this and for the previous like a set of generic tests that we want to run for some zombienet tests?
Let's try to prevent duplication.

Reply:
How do I do that? The tests all have different timeouts, I can't just put a function call in beforeAll or in T01...

Reply:
Very quick example, something like this: https://github.com/moondance-labs/tanssi/pull/1197